source/include/ota_http_private.h

```diff
@@ -23,6 +23,11 @@
  * http://www.FreeRTOS.org
  */
 
+/**
+ * @file ota_http_private.h
+ * @brief Contains function definitions of routines for OTA download using HTTP data plane.
```

Comment:
Maybe this might be better: _Contains definitions of routines for OTA download using HTTP data plane._ (Removed "function" since I believe routine and function are the same thing).
Not a blocker :)

Reply:
Thanks  https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/116/commits/3e5b1d96211d212a79fd5aa4a1bd33a51add1e85